README.md

```diff
@@ -5,134 +5,61 @@ Shopify API
-  ...
-end
-```
-is now
-```ruby
-ShopifyAPI::Session.temp(domain: domain, token: token, api_version: api_version) do
-  ...
-end
-```
-
-For example if you want to use the `2019-04` version you would create a session like this:
-```ruby
-session = ShopifyAPI::Session.new(domain: domain, token: token, api_version: '2019-04')
-```
-if you want to use the `unstable` version you would create a session like this:
-```ruby
-session = ShopifyAPI::Session.new(domain: domain, token: token, api_version: :unstable)
-```
-
-### Changes to how to define resources
-
-If you have defined or customized Resources, classes that extend `ShopifyAPI::Base`:
-The use of `self.prefix =` has been deprecated you should now use `self.resource =` and not include `/admin`.
-For example if you specified a prefix like this before:
-```ruby
-class MyResource < ShopifyAPI::Base
-  self.prefix = '/admin/shop/'
-end
-```
-You will update this to:
-```ruby
-class MyResource < ShopifyAPI::Base
-  self.resource_prefix = 'shop/'
-end
-```
-
-### URL construction
-
-If you have specifed any full paths for API calls in find
-```ruby
-def self.current(options={})
-  find(:one, options.merge(from: "/admin/shop.#{format.extension}"))
-end
-```
-would be changed to
-
-```ruby
-def self.current(options = {})
-  find(:one, options.merge(
-    from: api_version.construct_api_path("shop.#{format.extension}")
-  ))
-end
-```
-
-### URLs that have not changed
-
-- OAuth URLs for `authorize`, getting the `access_token` from a code, `access_scopes`, and using a `refresh_token` have _not_ changed.
-  - get: `/admin/oauth/authorize`
-  - post: `/admin/oauth/access_token`
-  - get: `/admin/oauth/access_scopes`
-- URLs for the merchant’s web admin have _not_ changed. For example: to send the merchant to the product page the url is still `/admin/product/<id>`
-
-## Usage
-
-### Requirements
-
-All API usage happens through Shopify applications, created by either shop owners for their own shops, or by Shopify Partners for use by other shop owners:
-
-* Shop owners can create applications for themselves through their own admin: https://docs.shopify.com/api/authentication/creating-a-private-app
+The Shopify API gem allows Ruby developers to access the admin section of Shopify stores programmatically.
+
+The best way to consume the Shopify API is through GraphQL, which enables high volume mutations, bulk operations, and access to all new features.
+
+The REST API is implemented as JSON over HTTP using all four verbs (GET/POST/PUT/DELETE). Each resource, like Order, Product, or Collection, has a distinct URL and is manipulated in isolation. In other words, we’ve tried to make the API follow the REST principles as much as possible.
+
+- [Shopify API](#shopify-api)
+- [Usage](#usage)
+  * [Requirements](#requirements)
+    + [Ruby version](#ruby-version)
+  * [Installation](#installation)
+  * [Getting Started](#getting-started)
+    + [1) Create an app](#1--create-an-app)
```

Comment:
All of these bookmarks don't link properly, there's an extra `-` after the number, should be `#1-create-an-app`

Reply:
Good catch, forgot about that. Thanks!